tests/vse_sync_pp/analyzers/test_gnss.py

```diff
@@ -0,0 +1,207 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Test cases for vse_sync_pp.analyzers.gnss"""
+
+from unittest import TestCase
+from collections import namedtuple
+from decimal import Decimal
+import math
+
+from vse_sync_pp.analyzers.gnss import (
+    TimeErrorAnalyzer
+)
+
+from .test_analyzer import AnalyzerTestBuilder
+
+TERR = namedtuple('TERR', ('timestamp', 'terror', 'state'))
```

Comment:
I think this format should be consistent with the expected format in the analyzer
```suggestion
TERR = namedtuple('TERR', ('timestamp', 'state', 'terror'))
```

Reply:
This consistency isn't required (and shouldn't be assumed): `TERR` only applies within this file.

The Analyzer's requirement is it is given objects with supported values for attributes `state` and `terror`. This test code does produce that.